src/gui/editors/PianoRoll.cpp

```diff
@@ -691,17 +671,15 @@ void PianoRoll::glueNotes()
 		QList<Note *> noteToRemove;
 
 		NoteVector::iterator note = selectedNotes.begin();
-		auto nextNote = note+1;
+		auto nextNote = note + 1;
 		NoteVector::iterator end = selectedNotes.end();
 
 		while (note != end && nextNote != end)
 		{
 			// key and position match for glue. The notes are already
 			// sorted so we don't need to test that nextNote is the same
 			// position or next in sequence.
-			if ((*note)->key() == (*nextNote)->key()
-				&& (*nextNote)->pos() <= (*note)->pos()
-				+ qMax(TimePos(0), (*note)->length()))
+			if ((*note)->key() == (*nextNote)->key() && (*nextNote)->pos() <= (*note)->pos() + qMax(TimePos(0), (*note)->length()))
```

Comment:
I think if we would do that, then we would split at every `&&` and `||` in the whole LMMS. I wonder if this is good.

Though the whole line looks a bit long now. Maybe we should try to change the line limit somehow. I set `ColumnLimit: 0` so that line break decisions in CTOR initializer lists are respected, but here, the human line break decisions somehow got ignored...

Reply:
Exactly that is done with `ColumnLimit: 120`.